### PR TITLE
Added pepinfo tutorial, A3, and supporting data

### PR DESCRIPTION
**Pull request by MikaelLeonard**

### CODE REVIEW

File: tools/PepInfo.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+A positive hydropathy index indicates that the segment is hydrophobic, while a negative hydropathy index indicates that the segment is hydrophilic. Understanding the polarity of segments in the viral protein sequence offers insights into their interactions, thereby contributing to a better understanding of the protein's structure.
+
+### 1. Navigate to [PepInfo](https://www.ebi.ac.uk/jdispatcher/seqstats/emboss_pepinfo)
+
+### 2. Input Protein Sequence
+In the "Input sequence" text area, paste or type the amino acid sequence of your protein of interest. Alternatively, you can choose a supported file from your computer or `use the example`, which will give you an amino acid sequence for human insulin-like growth factor protein complex. Here, we'll use the example data above (unknown virus related to the Locarnavirus sp.). 
+![input](img/pepinfo/Input_sequence.png)
+
+### 3. Select a Paremeter
```
Comment (ababaian) [inline tools/PepInfo.md]: "Select Parameters"

File: tools/img/.DS_Store
Comment (ababaian) [inline tools/img/.DS_Store:1]: Please remove all dot files from PR.

File: tools/toolTemplate.md
Comment (ababaian) [inline tools/toolTemplate.md:1]: Please do not delete the template file.

File: virus/Locarnavirus-Nudiformis.html
Comment (ababaian) [inline virus/Locarnavirus-Nudiformis.html:1]: HTML file is not required, pleaes only include markdown.

File: virus/virTemplate.md
Comment (ababaian) [inline virus/virTemplate.md:1]: please don't delete the template.

File: virus/Locarnavirus Nudiformis.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+# Discovering a New Locarnavirus Species: Exploring Genomic, Structural, and Functional Insights Redefines Current Definitions and Broadens Ecological Versatility and Host Range within Marine Metagenomes 
+written by: [Mikael Gouwtama](https://github.com/MikaelLeonard/VIRUSxDISCVRY.git)
```
Comment (ababaian) [inline virus/Locarnavirus Nudiformis.md]: remote the .git path, and simply make this reference your github.

Comment (ababaian) [reply virus/Locarnavirus Nudiformis.md]: so `https://github.com/MikaelLeonard`

File: tools/img/.DS_Store
Comment (MikaelLeonard) [reply tools/img/.DS_Store:1]: I tried removing all the dot DS files but I couldn't seem to find them in my computer.

File: .DS_Store
Comment (ababaian) [inline .DS_Store:1]: Please exclude dot files.

File: tools/img/.DS_Store
Comment (ababaian) [reply tools/img/.DS_Store:1]: You don't have to delete them (although you can on command line), but you should unstage them from your commit with `git rm`

File: virus/.DS_Store
Comment (ababaian) [inline virus/.DS_Store:1]: dot file

File: virus/img/.DS_Store
Comment (ababaian) [inline virus/img/.DS_Store:1]: dot file

File: virus/img/L.nudiformis/.DS_Store
Comment (ababaian) [inline virus/img/L.nudiformis/.DS_Store:1]: dot file

**Pull request merged**